dockerfile_functions.sh

```diff
@@ -308,15 +310,16 @@ RUN Write-Host ('Downloading {0} ...' -f \$env:JAVA_URL); \\
                 exit 1; \\
         }; \\
         \\
-        New-Item -ItemType Directory -Path C:\temp | Out-Null;
-
-RUN Write-Host 'Installing using MSI ...'; \\
+        New-Item -ItemType Directory -Path C:\temp | Out-Null; \\
+        \\
+        Write-Host 'Installing using MSI ...'; \\
         Start-Process -FilePath "msiexec.exe" -ArgumentList '/i', 'openjdk.msi', '/L*V', 'C:\temp\OpenJDK.log', \\
```

Comment:
Looks like we have tabs vs spaces for the above two commands.

Reply:
this should already be spaces

Reply:
On closer look it is all spaces, not sure why github seems to be highlighting them !